source/Playnite/GamesEditor.cs

```diff
@@ -467,6 +467,206 @@ public void OpenGameLocation(Game game)
             }
         }
 
+        public void CalculateGameSize(Game game, bool onlyIfDataMissing, bool updateGame = true, bool checkLastScanDate = false)

```

Comment:
updateGame should be image changed to something that says it update the actual library data, game object itself is always updated.

Reply:
Done as part of https://github.com/JosefNemec/Playnite/pull/3017/commits/3847f90b716c63d8f05b2025b6e5bdb8716c1adb